subprojects/shipkit/shipkit.gradle

```diff
@@ -8,8 +8,7 @@ apply plugin: 'com.gradle.plugin-publish'
 
 apply from: "$rootDir/gradle/upgrade-downstream.gradle"
 
-//TODO: WW uncomment when GradlePluginArtifactUrlResolver is released
-//apply plugin: 'org.shipkit.compare-publications'
+apply plugin: 'org.shipkit.compare-publications'
```

Comment:
I'd vote for making it the default and not applying it in the build.gradle file.
I just had a quick look at the code and I think `GradlePortalReleasePlugin` might be the place to apply the plugin.

Reply:
+1 let's include it by default!